PUBLISHING-AWS.md

```diff
@@ -12,6 +12,11 @@ You can also specify this in your `Infra.toml` file:
 regions = ["us-west-2", "us-east-1", "us-east-2"]
 ```
 
+Note: several commands work with AWS services, so there's some shared configuration related to AWS accounts and AWS IAM roles.
+For example, you can specify a role to assume before any API calls are made, and a role to assume before any API calls in a specific region.
+This can be useful if you want to use roles to control access to the accounts that own AMIs, for example.
+See the commented [example Infra.toml](tools/pubsys/Infra.toml.example) for details.
```

Comment:
Are you just updating the documentation to mention the possibility, or did something in the way the tools behaved suggest that you needed to configure a role to assume?

This is a relatively niche option, intended for the case where an otherwise unprivileged EC2 instance should be assuming roles in the same account, or a different account. Whether it's useful really depends on the scenario and the threat model.

Reply:
Good question: this is mostly mentioning the possibility since I ran into it directly in https://github.com/bottlerocket-os/bottlerocket/issues/2327

But *_this specific chunk_*[ I actually found and copy/pasted from `PUBLISHING.md`](https://github.com/bottlerocket-os/bottlerocket/blob/develop/PUBLISHING.md#configuring-the-publishing-process). It seemed like a good chunk to also utilize here in `PUBLISHING-AWS.md`. I agree, it is a bit of a niche use case, but I wasn't sure if it was common enough for people working on Bottlerocket to also mention here.